### PR TITLE
Add Dax's info to the team's page

### PR DESCRIPTION
**Pull request by daxmc99**
This adds my info to the team's page

### CODE REVIEW

File: company/team/index.md [hunk 1]
```diff
@@ -245,6 +245,13 @@ To add yourself to this page, copy the following template, paste it at the end o
 - [dave@sourcegraph.com](mailto:dave@sourcegraph.com), [GitHub](https://github.com/davejrt)
 - Dave likes riding his bike up hill, drinking coffee and travelling. After graduating University and getting his start as a Unix system administrator, he still has a soft spot for Solaris and enjoys solving problems on large scale infrastructure.
 
+## Dax McDonald (he/him) 
+
+- Software Engineer
+- Phoenix, AZ, USA 🇺🇸
+- [dax@sourcegraph.com](mailto:dax@sourcegraph.com), [Github](https://github.com/daxmc99), [Twitter](https://twitter.com/cloudmarooned)
+- Dax enjoys playing ultimate frisbee 🥏, running 🏃‍♂️ and running K8s on his Raspberry Pi. Before Sourcegraph, he worked on making Kubernetes easier to use with Rio and K3s.  He is an avid enthusiast of open-source and open-hardware. He has contributed to several open source projects such as Kubernetes, Linkerd & MetalLB
```
Comment (nicksnyder) [inline company/team/index.md:253]:
```suggestion
- Dax enjoys playing ultimate frisbee 🥏, running 🏃‍♂️ and running K8s on his Raspberry Pi. Before Sourcegraph, he worked on making Kubernetes easier to use with Rio and K3s. He is an avid enthusiast of open-source and open-hardware. He has contributed to several open source projects such as Kubernetes, Linkerd & MetalLB
```

Comment (daxmc99) [reply company/team/index.md]: Nice catch!
While this was an omission on my part, I'm curious what your personal beliefs are on having two spaces after a period are.

Comment (nicksnyder) [reply company/team/index.md]: As with any formatting discussion, the only meaningful argument is consistency within a given context. Within this file and our codebase I think single space after full stop is clearly the most consistent choice.

I also think single space after full stop is most consistent with the corpus of modern writing, but I have no interest in debating that opinion :)
https://twitter.com/profalankchen/status/1248661688099848192

**Comment by daxmc99** [comment]
This looks ready to merge 👍

**Comment by sqs** [comment]
@daxmc99 Go for it!

**Pull request merged**